openpype/pipeline/workfile/workfile_template_builder.py

```diff
@@ -1736,7 +1751,7 @@ def populate_create_placeholder(self, placeholder):
             failed = False
             self.create_succeed(placeholder, creator_instance)
 
-        self.cleanup_placeholder(placeholder, failed)
+        self.post_representation_load(placeholder, failed)
```

Comment:
It was named `cleanup_placeholder` beacuse `post_representation_load` in creat plugin does not make any sense.

Reply:
Ok, have you got a suggestion for a name?
"Clean up" has associations with removing and changing something.

Reply:
What do you think of `post_build_load()` which calls the method `cleanup_placeholder()`

Reply:
> "Clean up" has associations with removing and changing something.

But it did actually did cleanup placoholders when they are processed.

> What do you think of post_build_load() which calls the method cleanup_placeholder()

The `load` is not right, it does not have to load. I would suggest `finish_placeholder` or `post_process`.

Reply:
> Or `post_placeholder_process`?

👍

Reply:
`post_placeholder_process` sounds good to me. @ClementHector ?